stompjs/stompjs.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for stompjs 2.3.x
+// Project: https://github.com/jmesnil/stomp-websocket
+// Definitions by: Jimi (Dimitris) Charalampidis <https://github.com/jimic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+declare module 'stompjs' {
+
+    export const VERSIONS: {
+        V1_0: '1.0',
+        V1_1: '1.1',
+        V1_2: '1.2',
+        supportedVersions: () => '1.1,1.0'.split(',')
```

Comment:
I'm surprised this made it past the compiler! Definition files should not contain implementation. This appears to be equivalent to `supportedVersions(): string[]`. You could always type it as a tuple `["1.1", "1.0"]`, but I don't see why that would be useful.